svg/pservers/reftests/solidcolor-basic-001-ref.svg

```diff
@@ -0,0 +1,50 @@
+<svg id="svg-root"
+  width="100%" height="100%" viewBox="0 0 480 360"
+  xmlns="http://www.w3.org/2000/svg"
+  xmlns:xlink="http://www.w3.org/1999/xlink"
+  xmlns:html="http://www.w3.org/1999/xhtml">
+  <g id="testmeta">
+    <title>Hatch: Basic hatch.</title>
+    <html:link rel="copyright"
```

Comment:
Do we need individual copyright and license links, rather than a wpt-wide license and copyright?